website/core/center.js

```diff
@@ -9,11 +9,11 @@ const assign = require('object-assign');
 
 const center = React.createClass({
   render() {
-    let {style, ...props} = this.props;
-    style = assign({}, style, {textAlign: 'center'});
+    const {style, ...props} = this.props;
```

Comment:
Doing this to get all the code linted, this is an eslint error as we were previously reassigning to style.